### ISSUE TITLE
Volume level metrics for FSx for NetApp ONTAP missing in latest release

### ISSUE THREAD

**Issue by kurdekar** [NetApp/harvest#737]
In the latest release of Harvest, breaking down the dashboards for 7-mode and c-DOT seems is causing issues with Volume-level metrics for FSx for NetApp ONTAP. This was working fine in previous release. Modifying the query variable to label_values(datacenter) resolves this issue.

**Environment**
Provide accurate information about the environment to help us reproduce the issue.

- Harvest version: 21.11.0
- OS: Amazon Linux 2.
- Install method: CloudFormation Template as outlined here (https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/monitoring-harvest-grafana.html) and leverages steps from the ansible deployment outlined here:  https://www.github.com/netapp-automation/harvest_install
- ONTAP Version: 9.10

**To Reproduce**
Steps to reproduce the behavior: Deploy Harvest release 21.08.0. Both SVM and volume level metrics for FSx were visible on deployment.

**Expected behavior**
FSx SVM and Volume metrics should be visible.

**Actual behavior**
Volume level metrics are not reported in the dashboard. Modifying the query variable to label_values(datacenter) populates the volume level metrics

**Possible solution, workaround, fix**
Modifying the query variable to label_values(datacenter) populates the volume level metrics

**Comment by cgrinds** [comment]
If you have the changes, please attach to issue or email them. Thanks!

**Comment by cgrinds** [comment]
Thanks for reporting @kurdekar!

In `21.11`, `label_values` is used in four queries in `harvest_dashboard_volume.json` and
`harvest_dashboard_volume_details.json` for the template variables `Datacenter`, `Cluster`, `SVM`, and `Volume`

Two of those queries also include only label where `system_type!=\"7mode\"`

``` 
gron grafana/dashboards/cmode/harvest_dashboard_volume.json | rg "label_values|name"
json.templating.list[1].name = "Datacenter";
json.templating.list[1].query.query = "label_values(node_labels{system_type!=\"7mode\"}, datacenter)";
json.templating.list[2].name = "Cluster";
json.templating.list[2].query.query = "label_values(volume_labels{system_type!=\"7mode\",datacenter=\"$Datacenter\"}, cluster)";
json.templating.list[3].name = "SVM";
json.templating.list[3].query.query = "label_values(volume_labels{datacenter=\"$Datacenter\",cluster=\"$Cluster\"}, svm)";
json.templating.list[4].name = "Volume";
json.templating.list[4].query.query = "label_values(volume_labels{datacenter=\"$Datacenter\",cluster=\"$Cluster\",svm=~\"$SVM\"}, volume)";
```

Did you change the first two or all of them?
That should not be needed since FSx systems won't have 7-mode collectors running.
We'll try to reproduce with `21.11` monitoring an FSx system.

**Comment by kurdekar** [comment]
Just had to change the Datacenter variable for both "Volume" and "Volume -Details" dashboards to below:

Datacenter | label_values(datacenter)

**Comment by cgrinds** [comment]
Fixed by #740, will close after @kurdekar confirms

**Comment by cgrinds** [comment]
@kurdekar confirmed this fixed their issue. Closing
